src/sage/repl/preparse.py

```diff
@@ -1252,7 +1264,7 @@ def preparse_numeric_literals(code, extract=False, quotes="'"):
         sage: preparse_numeric_literals('3.14', quotes=None)
         'RealNumber(str().join(map(chr, [51, 46, 49, 52])))'
         sage: preparse_numeric_literals('5j', quotes=None)
-        'ComplexNumber(0, str().join(map(chr, [53])))'
+        'ComplexNumber(0, str().join(map(chr, [53])))'    
```

Comment:
```suggestion
        'ComplexNumber(0, str().join(map(chr, [53])))'
```

I think you accidentally added extra whitespace here while moving code around.